### ISSUE TITLE
Actually it's not 1kb

### ISSUE THREAD

**Issue by englishextra** [ruyadorno/simple-slider#32]
Please update your description, thanks

**Comment by ruyadorno** [comment]
hi @englishextra thanks for opening the issue, It will def help anyone with the same question.

The **npm** and **GitHub** descriptions are merely to help me market the module with other developers comparing different carousel libraries as there are [tons of them](https://npms.io/search?q=carousel). It was a [deliberate last-minute change](https://github.com/ruyadorno/simple-slider/commit/a150a87897f64e1888c9792c0e4174f8c73cf432) before releasing `v1.0.0`.

My rationale here is that the description is just a call to action and as soon the person is passed it and starts read the README file there's a big green badge that clarifies that the actual file size of the minified version after gzipping is 1.13kb. It is also stated multiple times in the README file and in the website so I don't think it's confusing in any way.

I agree that a more _exact_ description should read as:
- The `Math.floor(1.13)`kb JavaScript Carousel
- The 1kb-ish JavaScript Carousel
- The around 1kb JavaScript Carousel

But none of those don't sound as good as "The 1kb JavaScript Carousel".

In the spirit of clarification, here are the file size for each of the versions available:

- ![File Size](http://img.badgesize.io/ruyadorno/simple-slider/master/dist/simpleslider.js) - dist file
- ![File Size](http://img.badgesize.io/ruyadorno/simple-slider/master/dist/simpleslider.min.js) - dist file, minified
- ![File Size](http://img.badgesize.io/ruyadorno/simple-slider/master/dist/simpleslider.min.js?compression=gzip) -  Dist file, minified and gzipped

I hope that clarifies to you why the description reads like that and why I'm **not** going to change it.

Thanks again!

**TLDR;** It's just a marketing stunt, the real file size is stated everywhere else in the docs

**Comment by ruyadorno** [comment]
For anyone that might reach this one in the future, I'd like to state that I'm really open to accept PRs that would find smart ways to reduce the file size since this is one of the main features/goals of this lib.
